app/config/config.yml

```diff
@@ -145,3 +145,9 @@ jms_translation:
             output_dir: '%kernel.root_dir%/../vendor/ezsystems/ezplatform-admin-ui-modules/Resources/translations/'
             excluded_dirs: [Behat, Tests]
             output_format: "xliff"
+        page_builder:
+            dirs:
+                - '%kernel.root_dir%/../vendor/ezsystems/ezplatform-page-builder/src'
+            output_dir: '%kernel.root_dir%/../vendor/ezsystems/ezplatform-page-builder/src/bundle/Resources/translations/'
+            excluded_dirs: [Behat, Tests]
+            output_format: "xlf"
```

Comment:
Hmm, for other nodes we have `xliff`. If both are correct, please change it to be consistent with the others.

Reply:
I configured `output_format` as xlf because under page-builder repository we are using this extension: https://github.com/ezsystems/ezplatform-page-builder/tree/master/src/bundle/Resources/translations, which seems to be to used by SF as well:

* https://github.com/symfony/symfony/tree/3.4/src/Symfony/Component/Security/Core/Resources/translations
* https://github.com/symfony/symfony/tree/3.4/src/Symfony/Component/Form/Resources/translations

Reply:
I can work with both, but we need to be consistent at some point. I have to fix those in the [sync script](https://github.com/ezsystems/ezplatform-i18n/blob/master/bin/synchronize-translations.sh#L45), but it's not a big deal either.